### ISSUE TITLE
Alarming intermittent black screens when starting 1:1 video calls

### ISSUE THREAD

**Issue by matrixbot** [vector-im/vector-web#1894]
Created by @ matthew:matrix.org.

These keep biting Gary demos

**Comment by ara4n** [comment]
Need to repro and dig into the logs. @Manu - please yell if you see problems on Vector/Web where it just displays a black screen, presumably due to ICE failures.....

**Comment by ara4n** [comment]
see also #1960

**Comment by ara4n** [comment]
just had an epic trying to debug this between me & Amandine.   Her Chrome stopped issuing ICE candidates whilst on Amdocs seamless wifi (with the exception of two audio relay candidates on one call) - it looks like her WebRTC got wedged.  Was fixed after laptop reboot.

The only thing we can see going wrong is ignoring promises from .play() methods - stack traces like:

```
2016-08-31 11:10:19.457 Notifier.js:95 Uncaught (in promise) DOMException: The play() request was interrupted by a new load request.
_playAudioNotification @ Notifier.js:95
onRoomTimeline @ Notifier.js:238
```

and

```
2016-08-31 11:10:21.948 CallHandler.js:140 Call state in !lFdHwFlnCHQWDBxEjs:matrix.org changed to stop_ringing (ringing)
2016-08-31 11:10:21.949 CallHandler.js:140 Call state in !lFdHwFlnCHQWDBxEjs:matrix.org changed to connected (stop_ringing)
2016-08-31 11:10:21.949 index.js:116 newscreen room/!lFdHwFlnCHQWDBxEjs:matrix.org
2016-08-31 11:10:24.029 index.js:116 newscreen room/!lFdHwFlnCHQWDBxEjs:matrix.org
2016-08-31 11:10:24.511 call.js:899 Uncaught (in promise) DOMException: The play() request was interrupted by a new load request.
_tryPlayRemoteStream @ call.js:899
MatrixCall.setRemoteVideoElement @ call.js:202
showCall @ CallView.js:81onAction @ CallView.js:61
Dispatcher.$Dispatcher_invokeCallback @ Dispatcher.js:220
Dispatcher.dispatch @ Dispatcher.js:195
2016-08-31 11:10:24.833 call.js:947 Created answer: [object RTCSessionDescription]
```

so going to try to chain them rather than overlap them.

**Comment by ara4n** [comment]
hopefully fixed by matrix-org/matrix-js-sdk@b0782885d535cd44460e70f2895a39722fcb0d13
